frontend/components/songs/song_card.jsx

```diff
@@ -64,47 +64,48 @@ const SongCard = ({
         updateSelectedSong(song);
         if (!songCardDropdownState.isOpen) {
             placeSongCardDropdown(e);
-            updateSongCardDropdownState( {isOpen: true} );
-        } else if (dropdownMenuPointer === index) { // if user had clicked on the same SongCard, toggle dropdown
-            updateSongCardDropdownState( !songCardDropdownState.isOpen );
-        } else if (songCardDropdownState.isOpen && dropdownMenuPointer !== index) { // if user had clicked on a different SongCard, relocate the dropdown
-            updateSongCardDropdownState( {isOpen: false} );
+            updateSongCardDropdownState({ isOpen: true });
+        } else if (dropdownMenuPointer === index) {
+            // if user had clicked on the same SongCard, toggle dropdown
+            updateSongCardDropdownState(!songCardDropdownState.isOpen);
+        } else if (
+            songCardDropdownState.isOpen &&
+            dropdownMenuPointer !== index
+        ) {
+            // if user had clicked on a different SongCard, relocate the dropdown
+            updateSongCardDropdownState({ isOpen: false });
             placeSongCardDropdown(e);
-            updateSongCardDropdownState( {isOpen: true} );
-        };
+            updateSongCardDropdownState({ isOpen: true });
+        }
         updateDropdownMenuPointer(index);
         updateSelectedSong(song);
-    }
+    };
 
-    return ( 
+    return (
         <div className="song-card">
-            <div className="song-card-tracknum">
-                {tracknum}
-            </div>
+            <div className="song-card-tracknum">{tracknum}</div>
             <div className="song-card-title-artist-block">
-                <div className="song-card-title">
-                    {title}
-                </div>
+                <div className="song-card-title">{title}</div>
                 <div className="song-card-artist">
-                    {collabArtistNames}{songArtistName}
+                    {collabArtistNames}
+                    {songArtistName}
```

Comment:
Does this change the display if they are on separate lines?

Reply:
It does not